app/views/controllers/comments/_comment.erb

```diff
@@ -1,16 +1,21 @@
+<%# locals: (comment: nil, controls: @user, show_name: false) %>
```

Comment:
Makes sure `controls` is defined (below), but it should maybe default to `nil`. 

@mo-nathan @JoeCohen This is a new syntax for declaring locals in Rails partials and templates, and if it were Rubocoppable i'd be in favor of a cop on it. I have found it helpful and am implementing it piecemeal wherever i'm working.

Reply:
Yeah, I noticed it a few PRs back.  I like it.

Reply:
Defaulting to `@user` seems reasonable to me.